loadtest/setup_server.go

```diff
@@ -70,14 +70,38 @@ func SetupServer(cfg *LoadTestConfig) (*ServerSetupData, error) {
 			mlog.Error("Bulk-loading supports at most one plugin deployment at this time")
 		}
 
+		pluginId := "com.mattermost.sample-plugin"
 		plugin, err := os.Open("testfiles/com.mattermost.sample-plugin-webapp-only.tar.gz")
 		if err != nil {
 			return nil, err
 		}
 
+		mlog.Info(fmt.Sprintf("Checking if plugin is installed: %s", pluginId))
+		remotePlugins, resp := adminClient.GetPlugins()
+		if resp.Error != nil {
+			return nil, resp.Error
+		}
+		if len(remotePlugins.Active) > 0 {
+			mlog.Info(fmt.Sprintf("Plugin found, removing: %s", pluginId))
+			for _, remotePlugin := range append(remotePlugins.Active, remotePlugins.Inactive...) {
+				if remotePlugin.Id == pluginId {
+					status, resp := adminClient.RemovePlugin(pluginId)
+					if resp.Error != nil {
+						return nil, resp.Error
+					}
+					if status {
+						mlog.Info(fmt.Sprintf("Removing plugin: %s", pluginId))
```

Comment:
'removed' instead of 'removing'

Reply:
Wording updated in 41bb8dd84b2133d2a9920043fa99e02f74907b5e.